### PR TITLE
Update heroContents.html

### PR DESCRIPTION
**Pull request by CodeNinja-sys**

### CODE REVIEW

File: site/layouts/partials/homepage/heroContents.html [hunk 1]
```diff
@@ -4,7 +4,7 @@
         <h2 class="title">{{ .Site.Params.Tagline }}</h2>
         <div class="content">
             <p>yuzu is an experimental open-source emulator for the Nintendo Switch from the creators of Citra.</p>
-            <p>It is written in C++ with portability in mind, with builds actively maintained for Windows and Linux.</p>
+            <p>It is written in C++ with portability in mind, with builds actively maintained for Windows Linux and Android.</p>
```
Comment (FearlessTobi) [inline site/layouts/partials/homepage/heroContents.html:7]:
```suggestion
            <p>It is written in C++ with portability in mind, with builds actively maintained for Windows, Linux and Android.</p>
```

**Review by FearlessTobi — APPROVED** [review]

File: site/layouts/partials/homepage/heroContents.html (hunk 1 above)
Comment (CodeNinja-sys) [reply site/layouts/partials/homepage/heroContents.html:7]: > 

https://github.com/yuzu-emu/yuzu-emu.github.io/actions/runs/5156134202

**Comment by CodeNinja-sys** [comment]
https://github.com/yuzu-emu/yuzu-emu.github.io/actions/runs/5156134202

**Review by FearlessTobi — APPROVED** [review]

**Comment by FearlessTobi** [comment]
Thanks!

**Pull request merged**